lib/pure/math.nim

```diff
@@ -119,19 +138,28 @@ proc nextPowerOfTwo*(x: int): int {.noSideEffect.} =
 
 proc countBits32*(n: int32): int {.noSideEffect.} =
   ## Counts the set bits in `n`.
+  ##
+  ## .. code-block:: nim
+  ##  echo countBits32(13'i32) ## 3
   var v = n
   v = v -% ((v shr 1'i32) and 0x55555555'i32)
   v = (v and 0x33333333'i32) +% ((v shr 2'i32) and 0x33333333'i32)
   result = ((v +% (v shr 4'i32) and 0xF0F0F0F'i32) *% 0x1010101'i32) shr 24'i32
 
 proc sum*[T](x: openArray[T]): T {.noSideEffect.} =
-  ## Computes the sum of the elements in `x`.
+  ## Computes the sum of the elements in the array `x`.
```

Comment:
Nitpicking, but `openArray` is not limited to arrays, it can be (in this case) also a seq and (generally) a string.

I personally use "container" when describing `openArray`, but I don't know what is the official stance on this.

Reply:
Ok, I returned the description as it was before.